js/dropdown/shared/Multiselect.svelte

```diff
@@ -93,6 +93,16 @@
 		if (!allow_custom_value) {
 			input_text = "";
 		}
+
+		if (active_index !== null) {
+			add_or_remove_index(active_index);
+		} else {
```

Comment:
I don't think we should add this. If I hover over a choice, and then blur out, I would not expect that choice to be added to the list of selected choices.